### PR TITLE
Added button for add advisor modal

### PR DESCRIPTION
**Pull request by rbiz4**

### CODE REVIEW

File: static/js/redux/actions/modal_actions.jsx [hunk 1]
```diff
@@ -156,11 +156,20 @@ export const toggleSeparateAccountsModal = () => ({
...
 export const toggleSISImportDataModal = () => ({
   type: ActionTypes.TOGGLE_SIS_IMPORT_DATA_MODAL,
 });
+
+// ADD ADVISOR MODAL
+export const triggerAddAdvisorModal = () => ({
+  type: ActionTypes.TRIGGER_ADD_ADVISOR_MODAL,
+});
+
+export const toggleAddAdvisorModal = () => ({
```
Comment (Boccca2014) [inline static/js/redux/actions/modal_actions.jsx:173]: Duplicate, needs to be removed. Look at the full file to see the other value.

File: static/js/redux/constants/actionTypes.jsx [hunk 2]
```diff
@@ -191,3 +191,6 @@ export const CLOSE_TOS_MODAL = 'CLOSE_TOS_MODAL';
 
 export const TRIGGER_SIS_IMPORT_DATA_MODAL = 'TRIGGER_SIS_IMPORT_DATA_MODAL';
 export const TOGGLE_SIS_IMPORT_DATA_MODAL = 'TOGGLE_SIS_IMPORT_DATA_MODAL';
+
+export const TRIGGER_ADD_ADVISOR_MODAL = 'TRIGGER_ADD_ADVISOR_MODAL';
+export const TOGGLE_ADD_ADVISOR_MODAL = 'TOGGLE_ADD_ADVISOR_MODAL';
```
Comment (Boccca2014) [inline static/js/redux/constants/actionTypes.jsx:196]: Also a duplicate

File: static/js/redux/actions/modal_actions.jsx (hunk 1 above)
Comment (JJamesWWang) [reply static/js/redux/actions/modal_actions.jsx:173]: How is this a duplicate ?

Comment (JJamesWWang) [reply static/js/redux/actions/modal_actions.jsx:173]: Oh lol wait

**Review by JJamesWWang — CHANGES_REQUESTED** [review]
This is incomplete?

**Comment by rbiz4** [comment]
This is a WIP for now

**Comment by rbiz4** [comment]
Here is a flow of functionality (press the + button next to "Semester.ly - Advising Dashboard" to summon modal):
image
image
image

File: static/js/redux/ui/modals/add_advisor_modal.jsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            onChange={event => this.sendLastName(event)}
+          />
+          <p><br /></p>
+          <input
+            className="send-btn"
+            type="submit"
+            value="Send Info"
+            onClick={() => this.submitContent()}
+          />
+        </form>
```
Comment (JJamesWWang) [inline static/js/redux/ui/modals/add_advisor_modal.jsx:136]: Add some CSS:
image

We're presenting with this modal so it has to be up to that standard.

Comment (JJamesWWang) [reply static/js/redux/ui/modals/add_advisor_modal.jsx:136]: I would center the Send Info button, make it green, add some margins.
The textfields should expand so at least the text hints are complete (so there's no vertical scrollbar), I would make it extend all the way with some margins on the left and right. In other words, make it pretty

File: static/js/redux/ui/modals/add_advisor_modal.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+          accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          JhedId: this.state.jhed,
+          FullName: `${this.state.lastName},${this.state.firstName}`,
+        })
+      })
+        .then(() => this.setState({ jhed: this.state.jhed = '',
+          firsfirstNamet_name: this.state.firstName = '',
```
Comment (JJamesWWang) [inline static/js/redux/ui/modals/add_advisor_modal.jsx:76]: What happened here?

**Review by JJamesWWang — CHANGES_REQUESTED** [review]
Functionality wise LGTM, now add some CSS

**Comment by JJamesWWang** [comment]
You also have 8 lint errors, so fix those too.

File: static/js/redux/ui/modals/add_advisor_modal.jsx (hunk 4 above)
Comment (rbiz4) [reply static/js/redux/ui/modals/add_advisor_modal.jsx:76]: Spelling error, will correct.

**Comment by rbiz4** [comment]
Working on Lint errors, still WIP

**Review by Boccca2014 — APPROVED** [review]

**Comment by Boccca2014** [comment]
Rishi updated the CSS:
Screen Shot 2021-04-22 at 9 40 58 AM

**Review by JJamesWWang — APPROVED** [review]
LGTM

**Review by miaboloix — CHANGES_REQUESTED** [review]
Can we put that plus button in a better place?

**Comment by rbiz4** [comment]
New button position:
image

**Review by miaboloix — APPROVED** [review]

**Pull request merged**